Mathlib/Analysis/Fourier/Convolution.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2025 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+module
+
+public import Mathlib.Analysis.Distribution.FourierSchwartz
+public import Mathlib.Analysis.Convolution
+
+/-! # The Fourier transform of the convolution
+
+In this file we calculate the Fourier transform of a convolution.
+
+## Main statements
+* `Real.fourier_bilin_convolution_eq`: the Fourier transform of a convolution is the multiplication
+of the Fourier transform of the functions in terms of a general bilinear map.
+* `Real.fourier_smul_convolution_eq`: Variant for scalar multiplication.
+* `Real.fourier_mul_convolution_eq`: Variant for multiplication.
+
+-/
+
+@[expose] public section
+
+namespace Real
+
+variable {𝕜 R E F F₁ F₂ F₃ : Type*}
+
+open MeasureTheory Convolution
+
+variable [NontriviallyNormedField 𝕜] [NormedAddCommGroup E]
+  [NormedAddCommGroup F₁] [NormedAddCommGroup F₂] [NormedAddCommGroup F₃]
+  [InnerProductSpace ℝ E] [FiniteDimensional ℝ E] [MeasurableSpace E] [BorelSpace E]
+  [NormedSpace 𝕜 F₁] [NormedSpace 𝕜 F₂] [NormedSpace 𝕜 F₃]
+
+private theorem integrable_prod_sub (B : F₁ →L[𝕜] F₂ →L[𝕜] F₃) {f₁ : E → F₁} {f₂ : E → F₂}
+    (hf₁ : Integrable f₁) (hf₂ : Integrable f₂) (hf₁' : Continuous f₁) (hf₂' : Continuous f₂) :
+    Integrable (fun (p : E × E) ↦ ‖B‖ * (‖f₁ (p.1 - p.2)‖ * ‖f₂ p.2‖)) (volume.prod volume) := by
+  apply Integrable.const_mul
+  rw [integrable_prod_iff' (by measurability)]
+  constructor
+  · filter_upwards with x
+    exact (hf₁.comp_sub_right x).norm.mul_const _
+  have : Integrable (fun x ↦ ((∫ y, ‖f₁ y‖) * ‖f₂ x‖)) := by
+    apply hf₂.norm.bdd_mul (by measurability) (c := ‖(∫ y, ‖f₁ y‖)‖)
+    filter_upwards with; rfl
+  convert this using 1
+  ext x
+  simp_rw [norm_mul, norm_norm]
+  rw [integral_mul_const]
+  congr 1
+  convert integral_sub_right_eq_self _ x (μ := volume)
+  rfl
+
+open FourierTransform
+
+variable [NormedSpace ℂ F₃]
+
+/-- Calculate the Fourier transform of the convolution as a symmetric integral. -/
+theorem fourier_bilin_convolution_eq' (B : F₁ →L[𝕜] F₂ →L[𝕜] F₃) {f₁ : E → F₁} {f₂ : E → F₂}
```

Comment:
Just so that the name is more informative, as the statement is pretty different from the version without `'`.
```suggestion
theorem fourier_bilin_convolution_eq_integral (B : F₁ →L[𝕜] F₂ →L[𝕜] F₃) {f₁ : E → F₁} {f₂ : E → F₂}
```